front/app/components/admin/Graphs/LineBarChart/index.tsx

```diff
@@ -61,31 +66,39 @@ const LineBarChart = <Row,>({
     return <EmptyState emptyContainerContent={emptyContainerContent} />;
   }
 
-  const x = mapping.x;
-  const y = mapping.y[0];
-  if (typeof x === 'symbol' || typeof y === 'symbol') return null;
+  const { x, yBar, yLine } = mapping;
```

Comment:
It didn't really make sense to me that 'y' was an array, and that the mapping was used for the bar while the line was 'explicit' in the data structure. This way both are explicitly in the mapping and there's more type safety

Reply:
I agree on the naming not being clear. But this component always displays a cumulative series, so a total should always be present. The only variable thing is the bar that can have multiple values and create stacked bars.

Reply:
> But this component always displays a cumulative series, so a total should always be present. 
The `x` should also always be present, but it's still in the mapping- it's a required attribute there. That's how it currently works with all graphs. 

I also don't think we should tie the marks that they render to the type of data they're conveying: it doesn't always render a 'total', it renders lines and bars. The total just happens to be our use case.

> The only variable thing is the bar that can have multiple values and create stacked bars.
We're not currently using it to render stacked bars- we're just always grabbing the first value from the array, and that's fine, because it's sufficient for the current use case. I think that we should only make it an array when we actually start using it to render stacked bars

Reply:
all right cool, lets not rename then, will keep your change but `yBar` could be an array to generate a stacked bar like in the votes chart.

Reply:
Ah sorry, I didn't see that we do need a stacked bar in the case of votes! Yeah, sounds good then :+1: